tests/bugfixes/License.toml

```diff
@@ -0,0 +1,14 @@
+[license-file]
+comment = """Most images adapted from pngsuite with binary modification to reproduce
+specific faults.
+"""
+
+[[files]]
+file = "tests/bugfixes/gama-srgb-order-issue#304.png"
+SPDX-FileCopyrightText = "2013 zaimoku_woodpile <https://www.flickr.com/photos/11250735@N07/>"
+SPDX-License-Identifier = "CC-BY-2.0"
+Comment: """
+	Taken from: <https://www.flickr.com/photos/11250735@N07/8561945042>
+	with Modifications: Cropped, resized the image
+	Taken from: <https://raw.githubusercontent.com/slint-ui/slint/v0.1.6/examples/imagefilter/cat_preview_round.png>
+"""
```

Comment:
Is the syntax here correct? GitHub is highlighting it in red...

Reply:
I would [assume so](https://github.com/Randy8080/reference/blob/main/toml.md#multiline-string) and my editor is also highlighting this properly.

Reply:
Oops, that's why you're referring to. Yeah..